front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainsList.tsx

```diff
@@ -94,7 +94,7 @@ const ImportTrainScheduleTrainsList = ({
       dispatch(
         setSuccess({
           title: t('success'),
-          text: t('status.calculatingTrainScheduleCompleteAllSuccess', {
+          text: t('status.successfulImport', {
             trainsList,
             count: trainsList.length,
```

Comment:
For json imports, I think `trainList` will be empty (same for l.109)
```suggestion
            count: trainsList.length || trainsJsonData.length,
```